webextension/background.js

```diff
@@ -416,9 +406,9 @@ async function onMessage(message) {
       break;
     }
     case "requestScreenshot": {
-      browser.tabs.captureVisibleTab().then(screenshot => {
+      browser.tabs.captureTab(tabId).then(screenshot => {
```

Comment:
Can we pass in options to encode as `format: jpeg, quality: 75`? It's approximately what we do on the server side for webcompat.com.

This could be in a follow up bug as well.

Reply:
Might as well do it here, I suppose.